src/es6/SirenAction.js

```diff
@@ -271,7 +271,7 @@ export const performSirenAction = function(token, action, fields, immediate, byp
 /**
  * @param {string|Function} token
  * @param {() => Action} actionFactory
- * @param {FieldOverride[]} fieldOverrides
+ * @param {{ [key: string]: string }} fieldOverrides
```

Comment:
Are some places overriding fields with something other than `{ name: 'field-name', value: <some arbitrary value> }`? If so, that is technically [non-Siren](https://github.com/kevinswiber/siren?tab=readme-ov-file#name-1) 🤔

Reply:
It's just this method which takes an object